### PR TITLE
Add net45 dependency in nuspec.

### PR DESCRIPTION
**Pull request by kurtschelfthout**
See #410

### CODE REVIEW

**Comment by hickford** [comment]
What nuspec(s) does this produce? 

Reading [build.cmd], I see it [merges two packages using mergenupkg](https://github.com/fscheck/FsCheck/blob/master/build.fsx#L268)

File: src/FsCheck/paket.template [hunk 1]
```diff
@@ -25,6 +25,8 @@ description
     Properties are written in F#, C# or VB. FsCheck helps you to define properties, observe the distribution of test data, and define test data generators. Crucially, when a property fails, FsCheck automatically displays a minimal counter example.
 dependencies
     FSharp.Core >= 4.1.0
+	framework: net452
+		FSharp.Core >= 4.1.0
```
Comment (hickford) [inline src/FsCheck/paket.template]: Try FSharp.Core 4.1.18 here? Earlier versions of FSharp.Core pick up extra packages https://github.com/Microsoft/visualfsharp/issues/3451

Comment (kurtschelfthout) [reply src/FsCheck/paket.template]: Can't do that: 4.1.17 is the last FSharp.Core to have PCL builds.

Comment (hickford) [reply src/FsCheck/paket.template]: Fair enough.

**Pull request merged**